compiler/hlint-testing/data/report_template.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE HTML>
```

Comment:
Are we planning to use the HTML reports? At least initially I would focus on emitting regular diagnostics that we can show in the IDE instead of producing a separate report.

Reply:
No, no intent to use HTML reports. I think this is another template the library uses to generate `report.html` files from.